### PR TITLE
setting hearings-reform internal dns

### PR DESCRIPTION
**Pull request by stravze**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: environments/prod/hearings-reform-hmcts-net.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: "hearings.reform.hmcts.net"
```
Comment (timja) [inline environments/prod/hearings-reform-hmcts-net.yml:1]: can you not use the dev.platform.hmcts.net one like the rest of Shared services?

Comment (stravze) [reply environments/prod/hearings-reform-hmcts-net.yml:1]: I was trying to keep things uniform, so it doesn't confuse the Dev's in the future, please you say what you'd prefer ?

Comment (timja) [reply environments/prod/hearings-reform-hmcts-net.yml:1]: I would prefer that we use the same zone for everything if possible, using vh would linking the zone as well for each required env

could you try use dev.platform.hmcts.net please?

Comment (stravze) [reply environments/prod/hearings-reform-hmcts-net.yml:1]: of course, i'll get it sorted

Comment (stravze) [reply environments/prod/hearings-reform-hmcts-net.yml]: I have removed hearings and used dev.platform

File: environments/dev/dev-platform-hmcts-net.yml [hunk 2]
```diff
@@ -29,7 +29,14 @@ A:
...
     ttl: 300
+  - name: vh-user-api
+    ttl: 300
+    record:
+      - 10.145.15.250
 cname:
   - name: hmi-apim
     record: hmcts-dev.azurefd.net.
     ttl: 300
+  - name: vh-user-api
```
Comment (timja) [inline environments/dev/dev-platform-hmcts-net.yml:40]: why did you add the frontdoor entry? you just want the ingress controller

Comment (stravze) [reply environments/dev/dev-platform-hmcts-net.yml:40]: I add frontdoor into public dns then ?

**Review by timja — APPROVED** [review]

File: environments/dev/dev-platform-hmcts-net.yml (hunk 2 above)
Comment (timja) [reply environments/dev/dev-platform-hmcts-net.yml]: Internal apis don’t go on front door
